apps/api/src/app/subscribers/subscribers.controller.ts

```diff
@@ -476,12 +477,12 @@ export class SubscribersController {
   @ApiOperation({
     summary: 'Get in-app notification feed for a particular subscriber',
   })
-  @ApiOkPaginatedResponse(MessageResponseDto)
+  @ApiOkPaginatedResponse(FeedResponseDto)
```

Comment:
The feed notifications interface is slightly different from the message interface, that's why I introduced the new `FeedResponseDto` that has updated and correct fields and types